diff --git a/node_modules/react-helmet-async/index.d.ts b/node_modules/react-helmet-async/index.d.ts
index 3acc941..4e162e9 100644
--- a/node_modules/react-helmet-async/index.d.ts
+++ b/node_modules/react-helmet-async/index.d.ts
@@ -39,6 +39,7 @@ declare module 'react-helmet-async' {
     title?: string;
     titleAttributes?: Object;
     titleTemplate?: string;
+    children?: React.ReactNode;
   }
 
   export class Helmet extends React.Component<HelmetProps> {
